neural_modelling/makefiles/neuron/Makefile

```diff
@@ -38,9 +38,6 @@ MODELS = IF_curr_exp \
          IF_cond_exp_structural_last_neuron_distance_weight \
          IF_curr_exp_structural_last_neuron_distance_weight \
          IF_curr_exp_structural_random_distance_weight \
-         IF_cond_exp_stdp_mad_pair_additive_structural_last_neuron_distance_weight \
-         IF_cond_exp_stdp_mad_pair_additive_structural_random_distance_weight \
-         IF_curr_exp_stdp_mad_pair_additive_structural_random_distance_weight \
```

Comment:
Just a warning that if you're going to get rid of all structural plasticity + STDP, then you'll have to do something about the examples and integration tests that currently use it as well.

Reply:
will do. will make them skipped. coz we cant keep just going up compilers. and given this problem will be properly solveable with self partitioning. coz i bet the first thing we should do is make all structural ones do the compartmental stuff.